import-scripts/cvr_consent_status_checker.py

```diff
@@ -41,6 +41,11 @@ def fetch_expected_consent_status_values():
     for field,url in CVR_CONSENT_STATUS_ENDPOINTS.items():
         response = urllib.urlopen(url)
         data = json.loads(response.read())
+
+        if data['status'] != "SUCCESS" or len(data['cases']) == 0:
```

Comment:
should also check for status code

Reply:
also add a check (not ncessarily here) to see if total number of germline samples to remove is > than some threshold (can be arbitrary for now) - but above a certain threshold we send a warning and do not remove

Reply:
don't think we should just return none -- we should return the original struct 
{ PARTC_CONSENT: {}, PARTA_CONSENT:{}}

And then run a check on it during the main and give alerts afterwards